### PR TITLE
Bump version for oss_distillation_pipeline and chat_completion_pipeline

### PR DESCRIPTION
**Pull request by sharvin2187**
Updated the env to latest version

### CODE REVIEW

**Review by scravikiran — APPROVED** [review]
lgtm

File: assets/training/distillation/components/pipeline/spec.yaml [hunk 1]
```diff
@@ -259,7 +259,7 @@ jobs:
 
   oss_chat_completion_finetune:
     type: command
-    component: azureml:oss_chat_completion_finetune:0.0.19
+    component: azureml:oss_chat_completion_finetune:0.0.21
```
Comment (novaturient95) [inline assets/training/distillation/components/pipeline/spec.yaml:262]: lets update the data import one as well post oss ft component release

Comment (vikasagrawal-ms) [reply assets/training/distillation/components/pipeline/spec.yaml:262]: why are we jumping 2 versions?

Comment (sharvin2187) [reply assets/training/distillation/components/pipeline/spec.yaml:262]: ayush: holding on here based on offline discussions
vikas: distillation component was running an older version already, we are using 19 but the latest is 21

**Pull request closed without merging**